app/jobs/travel_to_work_area_and_london_borough_job.rb

```diff
@@ -0,0 +1,11 @@
+class TravelToWorkAreaAndLondonBoroughJob < ApplicationJob
+  queue_as :add_travel_to_work_area_and_london_borough
+
+  def perform(klass, id)
```

Comment:
This pattern was introduced because we needed to geocode both Sites and Providers. 
As we only need to add travel to work info to Sites then perhaps this can be simplified? Else @gpeng may want to leave as is as it's easily extendable? 🤷‍♂️

Reply:
right yea 🤦 Sorry there was some late night coding going on. It's a smart use of polymorphism 👌 
i don't think this would need to be extended so I'll simplify.